compile-protos-docker.sh

```diff
@@ -0,0 +1,61 @@
+# This script is used to generate Go and Python language bindings for the proto files in this repository.
+# It uses srl-labs/protoc:0.0.3 docker image to generate the bindings. The image contains the pinned versions of the
+# protoc compiler and the Go and Python GRPC plugins.
+
+DOCKER_CMD="docker run -v $(pwd):/in \
+  -v $(pwd)/proto:/in/github.com/openconfig/gnmi/proto \
+  ghcr.io/srl-labs/protoc:0.0.3"
```

Comment:
For my edification, how does this map to the `protoc` version that is in the container?

Reply:
@robshakir it doesn't direclty map to the protoc version, as there are more than just `protoc` in the package (namely grpc plugins for both py and go version).
Therefore, the idea is that `ghcr.io/srl-labs/protoc:0.0.3` is tagged in a semver fashion and the image tag matches the tag in the git repo for which the Dockerfile reveals the used versions: https://github.com/srl-labs/protoc-container/blob/v0.0.3/Dockerfile

Reply:
ACK - no problem. I was just wondering whether it is possible to directly map these together so that it's clearer, but it's written into the `pb.go` as to what versions are used so I don't see this is a huge issue.

The only downside here is that we stay pinned to that `0.0.3` version until an explicit change is made. If there's a semver convention here - could we consider tagging something like `1.x.x-latest`? (It'd be nice to stay close to HEAD for the generated files.)

Reply:
@robshakir good call, I see that the upstream changed the logic to include protoc__protoc-gen-go version in the image tag, so that it clear which sw components versions are used.

https://github.com/srl-labs/protoc-container#versioning

I have therefore update the branch in [4a11050](https://github.com/openconfig/gnmi/pull/136/commits/4a11050420ed9f7bd5a7f970134f2a2ccadfb855) to reflect that.

Now it is explicit which version of protoc and protoc-gen-go are used in the image.

Reply:
I can also port it to gnoi repo where it is even more of a pressing issue.